### DISCUSSION TITLE
File picker

### DISCUSSION THREAD

**Discussion by jmali86** [lxqt/lxqt#2482] [Q&A]
Is it possible to change the file picker GTK for kdialog portal from KDE adding some kind of variable and/or configuration?

Any help would be appreciate, thanks.

**Comment by stefonarch** [comment]
This is done by` xdg-desktop-portals` and one exist also for LXQt to use the its native filepicker:

https://github.com/lxqt/lxqt/wiki/ConfigMiscellaneous#lxqt-filedialog-in-gtk-applications

**Reply by tsujan** [reply]
And if your question is really about kdialog, you should ask it in a KDE forum.

**Reply by jmali86** [reply]
I'm using both, I just want to use the same file picker on my lxqt session but it seem no possible, even I asked on the distro forums and 'til today not an answer.

**Reply by jmali86** [reply]
steforarch thanks, I will try to modify some .config files from folder /usr/share/xdg-desktop-portal/portals/, also xdg-desktop-portal-lxqt was already installed, so far same results.
